Cartfile

```diff
@@ -1,8 +1,8 @@
 binary "https://www.mapbox.com/ios-sdk/MapboxAccounts.json" ~> 2.3.0
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.json" ~> 22.0
+binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.json" ~> 24.0
 binary "https://api.mapbox.com/downloads/v2/carthage/mobile-maps/mapbox-ios-sdk-dynamic.json" ~> 6.0
 binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon-ios.json" == 7.1.0
-github "mapbox/mapbox-directions-swift" ~> 1.0
+github "mapbox/mapbox-directions-swift" "18061abb9a185fb5200159eb757c1a81dd7656b6"
```

Comment:
Will we be moving to a specific version once the directions changes are merged into MapboxDirections?

Reply:
Correct. This is temporary while Directions didn't release with [corresponding update](https://github.com/mapbox/mapbox-directions-swift/pull/466).